rest/admin_api.go

```diff
@@ -358,14 +358,21 @@ func (h *handler) handleActiveTasks() error {
 func (h *handler) handleGetRawDoc() error {
 	h.assertAdminOnly()
 	docid := h.PathVar("docid")
-	includeDoc := h.getBoolQuery("include_doc")
-	redact := h.getOptBoolQuery("redact", true)
+
+	redact := h.getOptBoolQuery("redact", false)
 	salt := h.getQuery("salt")
 
+	var includeDoc bool
+
+	if redact && h.getQuery("include_doc") == "" {
```

Comment:
This logic feels a little awkward now.

I think we could make it a bit clearer if `getOptBoolQuery` returned the value, but also an `isSet` bool.

```golang
	includeDoc, includeDocSet := h.getOptBoolQuery("include_doc", true)
	redact, _ := h.getOptBoolQuery("redact", false)

	if redact && includeDocSet && includeDoc {
		return base.HTTPErrorf(http.StatusBadRequest, "redact and include_doc cannot be true at the same time. "+
			"If you want to redact you must specify include_doc=false")
	}

	// if include_doc has not been explicitly set and we're redacting, then override the default for include_doc
	if redact && !includeDocSet {
		includeDoc = false
	}
```